content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -19,7 +19,7 @@ fine grained authorization (such as separate views for Pod details and
 log retrievals), and can accept and serve those resources in different
 representations for convenience or efficiency.
 
-Kubernetes supports efficient change notifications on resources via *watches*.
+Kubernetes supports efficient change notifications on resources via *{{< glossary_tooltip text="watches" term_id="watch" >}}*.
```

Comment:
You can use the glossary definition here:
```suggestion
Kubernetes supports efficient change notifications on resources via
_watches_:
{{< glossary_definition prepend="in the Kubernetes API, watch is an" term_id="watch" length="short" >}}
```

Reply:
Alright. I'll add this.